### PR TITLE
AST-1462 - Wave accessibility issue for comment section

### PR DESCRIPTION
**Pull request by GauravKhupse**
### Description
Missing the comment label, wave accessibility issue, We need to add the <legend> tag in the <fieldset> for the comment section

### Screenshots

https://share.bsf.io/Jrun22NG

### How has this been tested?

Added in the card

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/markup-extras.php [hunk 1]
```diff
@@ -187,9 +187,9 @@ function astra_logo( $device = 'desktop', $echo = true ) {
 		$site_title           = astra_get_option( 'display-site-title-responsive' );
 		$display_site_title   = ( $site_title['desktop'] || $site_title['tablet'] || $site_title['mobile'] ) ? true : false;
 
-		$html = '';
-
+		$html            = '';
 		$has_custom_logo = apply_filters( 'astra_has_custom_logo', has_custom_logo() );
+		do_action( 'astra_before_logo_image', $device );
```
Comment (imnavanath) [inline inc/markup-extras.php:192]: This line is not relevant here, this was from Dark Mode branch. You can remove this in PR.

**Comment by imnavanath** [comment]
@GauravKhupse add changelog for this & make PR sync.

File: inc/markup-extras.php (hunk 1 above)
Comment (GauravKhupse) [reply inc/markup-extras.php]: Changes done!

File: changelog.txt [hunk 2]
```diff
@@ -1,5 +1,6 @@
 v3.8.0(Unreleased)
 - Improvement: Site Background will be applied as background CSS for Full-Width Contained/Stretched layouts instead of Content Background.
+- Fix:- Fixed legend tag missing Wave accessibility issue for comment section.
```
Comment (imnavanath) [inline changelog.txt:3]:
```suggestion
- Fix: W3 validation - legend HTML tag missing as fieldset tag is being used for comment section.
```

Comment (GauravKhupse) [reply changelog.txt]: Changes done!

File: assets/css/unminified/main.css [hunk 3]
```diff
@@ -3243,3 +3243,8 @@ input[type="color"]::-webkit-color-swatch {
 .ast-flex-grow-1 {
   flex-grow: 1;
 }
+
+.comment-form-legend {
+  margin-bottom: unset;
+  padding: 0 0.5em;
+}
```
Comment (imnavanath) [inline assets/css/unminified/main.css:3250]: Will this override after grunt minify run?

Comment (GauravKhupse) [reply assets/css/unminified/main.css:3250]: I have updated this code in main.scss file to override this code.

**Review by imnavanath — APPROVED** [review]

**Comment by Nikschavan** [comment]
@imnavanath / @premanshup We should also check why our automated test was not able to report this.

We use [this package](https://www.npmjs.com/package/@wordpress/jest-puppeteer-axe) which has many customization options for report levels

**Comment by premanshup** [comment]
> @imnavanath / @premanshup We should also check why our automated test was not able to report this.
> 
> We use [this package](https://www.npmjs.com/package/@wordpress/jest-puppeteer-axe) which has many customization options for report levels

@imnavanath Please check this ^

**Comment by Nikschavan** [comment]
FYI - I don't mean for this to be a blocker for merging this PR

File: sass/main.scss [hunk 4]
```diff
@@ -76,3 +76,8 @@
 # Content
 --------------------------------------------------------------*/
 @import "site/main";
+
+.comment-form-legend {
+	margin-bottom: unset;
+	padding: 0 0.5em;
+}
```
Comment (premanshup) [inline sass/main.scss:83]: @GauravKhupse We can move this CSS to the dynamic CSS for optimization:
`wp-content\themes\astra\inc\dynamic-css\comments.php`. Please check.

Comment (GauravKhupse) [reply sass/main.scss]: Changes done!

**Comment by premanshup** [comment]
> > @imnavanath / @premanshup We should also check why our automated test was not able to report this.
> > We use [this package](https://www.npmjs.com/package/@wordpress/jest-puppeteer-axe) which has many customization options for report levels
> 
> @imnavanath Please check this ^

@imnavanath Can you please add a task for this?

**Comment by imnavanath** [comment]
> @imnavanath Can you please add a task for this?

Added it here @premanshup - https://wp-astra.atlassian.net/browse/AST-1492

**Review by premanshup — APPROVED** [review]

**Pull request merged**